### ISSUE TITLE
rangeUntil extension function import marked as unused

### ISSUE THREAD

**Issue by dalewking** [pinterest/ktlint#2373]
I have an extension function like this to allow you to say something like `now..<5.minutes` to get an open ended range of instants:
```
/**
 * Convenience method to convert an instant and a duration into an open-ended range of instants. For
 * example: `now..<5.minutes` produces an open-ended range from now until 5 minutes from now
 *
 * @receiver The starting instant of the range
 * @param duration The amount of time to add to the starting time to get the end of the range
 * @return An open-ended range that starts at `this` and ends before `this + duration`
 */
operator fun Instant.rangeUntil(duration: Duration): OpenEndRange<Instant> = this..<this + duration
```

## Expected Behavior
When I add an import in another file to use this operator, ktlint should know that ..< corresponds to the rangeUntil function name and should not report the import of the rangeUntil name as unused

## Observed Behavior
Ktlint reports an unused import on the rangeUntil operator

## Steps to Reproduce
Define a rangeUntil extension operator function in one file and import it and use it in another file

**Comment by dalewking** [comment]
Ktlint is smart enough to do this for rangeTo function and the .. operator.

**Comment by dalewking** [comment]
I believe all that is needed is adding rangeUntil to this line: https://github.com/pinterest/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/rules/NoUnusedImportsRule.kt#L294

**Comment by paul-dingemans** [comment]
Tnx for suggestion. It was an easy fix.
